web/src/main/java/com/bazaarvoice/emodb/web/resources/queue/QueueResource1.java

```diff
@@ -341,6 +351,36 @@ public SuccessResponse purge(@QueryParam("partitioned") BooleanParam partitioned
         return SuccessResponse.instance();
     }
 
+    @PUT
+    @Path("/UpdateParameterStore")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @ApiOperation (value = "update param operation at aws parameter store .",
+            notes = "Returns a SuccessResponse.", response = SuccessResponse.class)
+    public SuccessResponse updateParam(Map<String, String> keyValuePair) {
+        String key = keyValuePair.keySet().iterator().next();
+        String value = keyValuePair.get(key);
+
+        Long update_version = _parameterStoreUtil.updateParameter(key, value);
+        return SuccessResponse.instance().with(ImmutableMap.of("status", "200 | ssm-parameter updated successfully, update_version: " + update_version));
+    }
+
+    @PUT
+    @Path("/QueueExecutionAttributes/{queue_type}/{queue_name}")
```

Comment:
suggestion: should not the path be `/{queue_type}/{queue_name}/QueueExecutionAttributes` ?

Reply:
Modified